pages/Favorites/subscriptions.js

```diff
@@ -22,9 +24,13 @@ export default function favorites(subscribe) {
         actionOnClick: addFavorites(action.productId, true),
         message: 'favorites.removed',
         replaceable: true,
-        duration: 6000,
+        duration: 2500,
       }));
     }, FAVORITES_SHOW_TOAST_DELAY);
   });
+
+  subscribe(routeDidLeave(FAVORITES_PATH), ({ dispatch }) => {
```

Comment:
Please set the toasts to hide whenever a user changes route. Not when they leave the favorites route.

Reply:
ok will do as you wish. but doesn't this violate the concept of toast anyways?